CHANGELOG.md

```diff
@@ -70,26 +71,30 @@ Note, since the pipeline is now using Nextflow DSL2, each process will be run wi
 
 | Dependency      | Old version | New version |
 | --------------- | ----------- | ----------- |
+| `asciigenome`   | 1.16.0      |             |
```

Comment:
no update?

Reply:
it was removed

Reply:
in #491 after voting its use in viralrecon channel. No one was using it.